app/services/os-datastore.js

```diff
@@ -493,3 +498,20 @@ module.exports.publish = function(descriptor, options) {
       });
   });
 };
+
+module.exports.isDataStoreUrl = function(urlToCheck) {
+  if (!_.isString(urlToCheck)) {
+    return false;
+  }
+  var parsed = url.parse(urlToCheck);
+  if (!_.isObject(parsed)) {
+    return false;
+  }
+
+  var checkHost1 = parsed.hostname == 's3.amazonaws.com';
```

Comment:
Okay, this is the new endpoint:
`http://staging.openspending.org/datastore/info`

You can read about it in the [docs](http://docs.openspending.org/en/latest/developers/conductor/)